addons/account/migrations/11.0.1.1/openupgrade_analysis_work.txt

```diff
@@ -0,0 +1,193 @@
+account      / account.journal          / active (boolean)              : NEW 
+account      / account.journal          / bank_statements_source (selection): selection_keys is now '['manual', 'undefined']' ('['manual']')
+account      / account.journal          / color (integer)               : NEW 
+account      / account.journal          / display_on_footer (boolean)   : DEL 
+# NOTHING TO DO
+
+account      / account.move             / statement_line_id (many2one)  : DEL relation: account.bank.statement.line
+account      / account.move.line        / statement_line_id (many2one)  : NEW relation: account.bank.statement.line
+# DONE: post-migration: move statement_line_id values from account.move to account.move.line
+
+account      / account.payment          / message_follower_ids (one2many): NEW relation: mail.followers
+account      / account.payment          / message_ids (one2many)        : NEW relation: mail.message
+account      / account.payment          / message_last_post (datetime)  : NEW 
+account      / account.payment          / state (selection)             : selection_keys is now '['cancelled', 'draft', 'posted', 'reconciled', 'sent']' ('['draft', 'posted', 'reconciled', 'sent']')
+account      / account.payment          / website_message_ids (one2many): NEW relation: mail.message
+account      / account.payment          / writeoff_label (char)         : NEW 
+account      / account.payment.term     / sequence (integer)            : NEW required: required, req_default: function
+account      / account.reconcile.model.template / chart_template_id (many2one)  : NEW relation: account.chart.template, required: required
+account      / account.tax.template     / cash_basis_account (many2one) : NEW relation: account.account.template
+account      / account.tax.template     / tax_exigibility (selection)   : NEW selection_keys: ['on_invoice', 'on_payment']
+account      / res.company              / account_opening_move_id (many2one): NEW relation: account.move
+account      / res.company              / account_setup_bank_data_done (boolean): NEW 
+account      / res.company              / account_setup_bar_closed (boolean): NEW 
+account      / res.company              / account_setup_coa_done (boolean): NEW 
+account      / res.company              / account_setup_company_data_done (boolean): NEW 
+account      / res.company              / account_setup_fy_data_done (boolean): NEW 
+account      / res.company              / tax_exigibility (boolean)     : NEW 
+account_tax_cash_basis / account.tax              / use_cash_basis (boolean)      : selection_keys is now '['on_invoice', 'on_payment']' ('False')
+account_tax_cash_basis / account.tax              / use_cash_basis (boolean)      : type is now 'selection' ('boolean')
+account_tax_cash_basis / account.tax              / use_cash_basis (boolean)      : was renamed to tax_exigibility [nothing to do]
+# NOTHING TO DO
```

Comment:
looks like we need to make sure that tax_exigibility is set to 'on_payment' if use_cash_basis was set to true, and 'on_invoice' otherwise.

Reply:
Set the company's tax_exigibility to True if any of it's taxes was set to use_cash_basis as well.